update node version support in package.json (#2998)
AzureAD/microsoft-authentication-library-for-js: lib/msal-node/README.md

```javascript
npm install @azure/msal-node
```
## Node Version Support

> MSAL Node will follow the [Long Term Support (LTS) schedule of the Node.js project](https://nodejs.org/en/about/releases/). Our support plan is as follows.

Reviewer: ```suggestion
MSAL Node will follow the [Long Term Support (LTS) schedule of the Node.js project](https://nodejs.org/about/releases/). Our support plan is as follows.
```

Any major MSAL Node release:

> - Will support active LTS versions

Reviewer: ```suggestion
- Will support current LTS and active versions
```

> - Will support Maintenance LTS, Active LTS, and Current versions of Node

Reviewer: ```suggestion
- Will support Maintenance LTS and Active LTS versions of Node (even-number releases)
```

> - Will support Maintenance LTS, Active LTS, and Current versions of Node

Reviewer: ```suggestion
- Will support stable (even-numbered) Maintenance LTS, Active LTS, and Current versions of Node
```

- Will drop support for any previously supported Node versions that have reached end of life
- Will not support prerelease/preview/pending versions until they are stable

> | MSAL Node version | MSAL support status     | Supported Angular versions |

Reviewer: ```suggestion
| MSAL Node version | MSAL support status     | Supported Node versions |
```

Author: oops!

|-------------------|-------------------------|----------------------------|

> | 1.x.x             | Active development      | 10(maintanence), 12, 14    |

Reviewer: ```suggestion
| 1.x.x             | Active development      | 10, 12, 14, 15    |
```

I don't think you need to label versions as maintenance, as the Node.js page details that.

Author: We do not want to list odd numbers, as per the norm??

Reviewer: Ah okay, then we need to adjust the line above and the `engines` field.

Reviewer: Suggestion updated

Author: What I think is this, we will support the range in `engines` since it is usually in between for any node updates, but specify we may not upgrade MSAL Node for odd numbered Node.js updates. I haven't made up my mind yet, we can discuss this and resolve offline.

Author: As discussed offline, updating to even versions for now. Let's wait on the feedback and update as needed.
